pudb/debugger.py

```diff
@@ -2000,6 +2045,12 @@ def redraw_screen(w, size, key):
         def help(pages):
             self.message(pages, title="PuDB - The Python Urwid Debugger")
 
+        def edit_current_frame(w, size, key):
+            file_name = self.debugger.curframe.f_code.co_filename
+            line_number = self.debugger.curframe.f_lineno
+            open_file_editor(file_name, line_number)
+            redraw_screen(w, size, key)
```

Comment:
Elsewhere, I thought we got rid of `redraw_screen`?

Reply:
Sorry I missed this, and removed only 1 from another place.
DONE